### PR TITLE
Create dependencies.R

### PR DESCRIPTION
**Pull request by meghalithic**
@johnbradley is this what you were talking about?

### CODE REVIEW

**Comment by johnbradley** [comment]
> @johnbradley is this what you were talking about?

Yes. This way an R user can just run this script to setup their requirements. We could also potentially use it to build a container. You could also consider using `devtools::install_version`: https://www.rdocumentation.org/packages/devtools/versions/1.13.6/topics/install_version

File: Scripts/dependencies.R [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+##versions:
+#devtools 2.4.3
+#rjson 0.2.21
+#tidyr 1.2.0
+#dplyr 1.0.8
+#ggplot2 3.3.5
+#RColorBrewer 1.1.2
+#stringr 1.4.0
+
+install.packages(c("devtools", "rjson", "tidyr", "dplyr", "ggplot2", "RColorBrewer", "stingr"))
```
Comment (johnbradley) [inline Scripts/dependencies.R]: Should `stingr` be `stringr`?

Comment (johnbradley) [reply Scripts/dependencies.R]:
```
> install.packages(c("devtools", "rjson", "tidyr", "dplyr", "ggplot2", "RColorBrewer", "stingr"))
Warning in install.packages :
  package ‘stingr’ is not available for this version of R
```

Comment (meghalithic) [reply Scripts/dependencies.R]: Great catch!

**Comment by meghalithic** [comment]
I'm not sure how to 1) install a certain version of devtools and 2) install multiple packages while specifying their version. I haven't found anything on stackoverflow.

**Comment by johnbradley** [comment]
> I'm not sure how to 1) install a certain version of devtools and 2) install multiple packages while specifying their version. I haven't found anything on stackoverflow.

I don't think there is a way to install a specific version of devtools since devtools supplies the `install_version` function.
You could do something like this though:
```
install.packages("devtools")
devtools::install_version("rjson", version="0.2.21")
devtools::install_version("tidyr", version="1.2.0")
devtools::install_version("dplyr", version="1.0.8")
devtools::install_version("ggplot2", version="3.3.5")
devtools::install_version("RColorBrewer", version="1.1.2")
devtools::install_version("stringr", version="1.4.0")
```
I can create a container with the versions of software you prefer so even though we can't easily control the version of devtools we can make it available.

**Comment by cflorian1** [comment]
@johnbradley and @megbalk for managing dependencies, would something like the renv package be useful? We currently use this at NEON in the data processing R code and have had good success.

**Comment by johnbradley** [comment]
@cflorian1 Thanks for this suggestion. I wasn't aware of `renv`. It looks really promising.

**Comment by meghalithic** [comment]
@johnbradley I am going to try to chat with someone at NEON who knows how to create a renv as it seems a bit beyond my skill level.

**Comment by johnbradley** [comment]
@megbalk Are you still working on these changes or are they ready for review?

**Pull request merged**